frontend/src/components/Common/ReportDialog/index.tsx

```diff
@@ -150,7 +192,11 @@ const ReportDialog = memo(
           </span>
           <Button className={classes.actionButton} variant="outlined">
             <PDFDownloadLink document={reportDoc} fileName={getPDFName}>
-              {({ loading }) => (loading ? 'Loading document...' : 'Download')}
+              {({ loading }) =>
+                loading || documentIsLoading
+                  ? 'Loading document...'
```

Comment:
These texts should get translated if possible

Reply:
Updated